client/my-sites/patterns/types.ts

```diff
@@ -12,7 +9,15 @@ export type RouterContext = Context & {
 	queryClient: QueryClient;
 };
 
-export type { Category, Pattern };
+export type { Pattern };
+
+export type Category = {
+	name: string;
+	title: string;
+	description: string;
+	pattern_count: number;
+	page_pattern_count: number;
```

Comment:
Looking at these properties in isolation it's difficult to understand the distinction between the two. The `page` term is also a bit misleading considering that it usually refers to pagination. What do you think of the following?

```suggestion
	regular_pattern_count: number;
	page_pattern_count: number;
```

I like the following better but I understand this may just be personal preference:

```suggestion
	number_of_regular_patterns: number;
	number_of_page_patterns: number;
```

Reply:
Also, what do you think of converting those properties to camelCase so we have the same notation everywhere in our client code? I think we could take advantage of the `snakeToCamelCase()` [function ](https://github.com/Automattic/wp-calypso/blob/888686f8fc26ad5bd2affd822dfeed609668929c/packages/js-utils/src/snake-to-camel-case.ts) and [transform the response](https://tkdodo.eu/blog/react-query-data-transformations) in our `react-query` code.

Reply:
That's fair 👍 I've tweaked the hook to return snake cased object keys.